### DISCUSSION TITLE
Welcome to redwoodjs-stripe Discussions!

### DISCUSSION THREAD

**Discussion by chrsnymn** [chrsnymn/redwoodjs-stripe#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adriatic** [comment]
### Hello @chrisvdm 

A brief resume - please check https://www.linkedin.com/in/nikolaj-ivancic-6ab948a8/ (full resume would take too long, it's sufficient to say that I started my US career 1982, in Mountain View, CA

I liked the idea of Redwood Applications integration with Stripe for some time. Properly stated I like any work that would make RW even more prominent.

I plan to use this [item](https://github.com/chrisvdm/redwoodjs-stripe/discussions/2#discussioncomment-2851734) as my **Diary**, to keep everyone updated about my own progress. If you find a better place to keep my updates (wiki?) please do let me know.

Lastly my current (large) RW project is [RW Cookbook](https://community.redwoodjs.com/t/redwood-applications-development-cookbook/3280). Perhaps you may consider contributing to that cookbook by authoring the "recipe" how this project got finished
